distribution/sdk/sample-nsili-server/pom.xml

```diff
@@ -137,6 +137,9 @@
                             mina-core,
                             ftpserver-core
                         </Embed-Dependency>
+                        <Import-Package>
+                            *;resolution:=optional
```

Comment:
❓ Is this required because there were some unavailable packages it was trying to resolve?

Reply:
Yeah the new bundle plugin does a much better job of determining imports, but most of the stuff it wants to import is for functionality that we just don't use. So being that this is a test only module, I just didn't want to spend hours trying to figure out exactly which imports are legit and which ones are bogus.